README.md

```diff
@@ -1 +1 @@
-# tictactoe-java
+# tictactoe-java [![Build Status](https://travis-ci.org/klkelley/tictactoe-java.svg?branch=skeleton-setup)](https://travis-ci.org/klkelley/tictactoe-java)
```

Comment:
Nice. I'd also like to see instructions here on how to do common tasks like run the tests, build the project, run the project, etc.